wolox-training/mr-rails
# Inline editor

# Summary
* Added Versionist dependency
* Added REST API for updating articles
* Added inline editor for articles in /articles

# Test plan
1. Go to /articles
2. Click on the 'Edit' link next to and article
3. The inline editor should be displayed
4. Edit the article and click on "Save"
5. The article should be updated

# Trello card
https://trello.com/c/XrwGkUFW/23-crear-update-de-articulos-con-javascript
